src/components/Banner/Banner.module.css

```diff
@@ -1,73 +1,137 @@
-.bannerDialog {
-  @apply w-full p-8 rounded border shadow-1;
-  @apply relative;
-  @apply grid gap-4;
-
+/* stylelint-disable comment-whitespace-inside, selector-nested-pattern, declaration-empty-line-before */
```

Comment:
We seem to have style linting on `main` that conflicts with how we're writing CSS on `next`. 🤷

Reply:
ahh shoot, yeah I think we're using `config-standard` on main but `config-recommended` on next 🤔  i'll make a note to update `main`

Reply:
https://github.com/chanzuckerberg/edu-design-system/pull/1028 -- I didn't realize this hadn't merged yet, so you might need to merge new changes again 😅  then these suppressions shouldn't be needed